client/file_logging.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+
+import king_phisher
+import king_phisher.client.plugins as plugins
+import king_phisher.client.gui_utilities as gui_utilities
+
+# logger name value
+LOGGER_NAME = ''
+
+class Plugin(plugins.ClientPlugin):
+	authors = ['Zach Janice']
+	title = 'Logger'
+	description = """
+	Keep logs of campaign feedback and results. The directory
+	of the logged file(s), the file size (in MB) of the log, and
+	the number of log files kept can be specified. Plugin requires
+	the 'logging' and 'os' packages available for Python.
+	"""
+	req_packages = {}
+	homepage = 'https://github.com/securestate/king-phisher-plugins'
+	options = []
+
+	# this is the primary plugin entry point which is executed when the plugin is enabled
+	def initialize(self):
+		# ensure the directory for the logs exists
+		log_dir = king_phisher.client.application.USER_DATA_PATH
```

Comment:
The `king_phisher.client.application` module should explicitly be imported so this can safely be accessed.